<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>670</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>670</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>670</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Menu</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Zdjecia.qrc">
    <normaloff>:/new/prefix1/Zdjecia/F1.svg.png</normaloff>:/new/prefix1/Zdjecia/F1.svg.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>651</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Menu</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>241</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Formula1</family>
       </font>
      </property>
      <property name="text">
       <string>Obecny sezon</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>241</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Formula1</family>
       </font>
      </property>
      <property name="text">
       <string>Następny wyścig</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>241</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Formula1</family>
       </font>
      </property>
      <property name="text">
       <string>*coming soon*</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>241</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Formula1</family>
       </font>
      </property>
      <property name="text">
       <string>Oficjalne media F1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>40</y>
        <width>561</width>
        <height>231</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="Zdjecia.qrc">:/new/prefix1/Zdjecia/F1500.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>300</y>
        <width>311</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Formula1</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Następny wyścig:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>360</y>
        <width>331</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Formula1</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>GP Meksyku</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>420</y>
        <width>311</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Formula1</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Do następnego wyścigu:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>500</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Formula1</family>
       </font>
      </property>
      <property name="text">
       <string>Autor</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Kierowcy</string>
     </attribute>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>111</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Lewis Hamilton</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>111</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Valtteri Bottas</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>111</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Max Verstappen</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>111</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Lando Norris</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>111</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Serigo Perez</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>111</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Daniel Riccardo</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>111</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Sebastian Vettel</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>111</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Lance Stroll</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>111</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Carlos Sainz Jr.</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>111</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Charles Leclerc</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>111</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Fernando Alonso</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>111</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Esteban Ocon</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>390</y>
        <width>111</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Pierre Gasly</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>420</y>
        <width>111</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Yuki Tsunoda</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>450</y>
        <width>111</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Kimi Raikkonen</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_16">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>480</y>
        <width>131</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Antonio Giovinazzi</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_17">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>510</y>
        <width>111</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>George Russell</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_18">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>540</y>
        <width>111</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Nicholas Latifi</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_19">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>570</y>
        <width>121</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Mick Schumacher</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_20">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>600</y>
        <width>111</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Nikita Mazepin</string>
      </property>
     </widget>
     <widget class="QLabel" name="imie_label">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>70</y>
        <width>411</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>30</y>
        <width>361</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Imię i Nazwisko:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>130</y>
        <width>361</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Zespół:</string>
      </property>
     </widget>
     <widget class="QLabel" name="zespol_label">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>180</y>
        <width>411</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>230</y>
        <width>361</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Wygranych wyścigów:</string>
      </property>
     </widget>
     <widget class="QLabel" name="wygrane_label">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>270</y>
        <width>411</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>320</y>
        <width>361</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Przejechanych wyścigów:</string>
      </property>
     </widget>
     <widget class="QLabel" name="przejechane_label">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>370</y>
        <width>411</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>430</y>
        <width>361</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Suma punktów w tym sezonie:</string>
      </property>
     </widget>
     <widget class="QLabel" name="sumapkt_label">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>480</y>
        <width>411</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Archiwum</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Zdjecia.qrc"/>
  <include location="Zdjecia.qrc"/>
 </resources>
 <connections/>
</ui>
